stansteel-and-hotmix/models/sale_order_line.py

```diff
@@ -0,0 +1,14 @@
+from odoo import models, fields, api
+
+
+class SaleOrderLine(models.Model):
+    _inherit = 'sale.order.line'
+
+    product_cost = fields.Float(string="Cost", readonly=True, compute="_compute_cost")
```

Comment:
why not use the standard price directly in the computation?
I thinks the field is unnecessary.